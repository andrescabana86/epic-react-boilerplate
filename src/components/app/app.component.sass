@import "src/style/colors"
.app
  color: $green