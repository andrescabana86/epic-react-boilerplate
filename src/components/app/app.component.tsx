import React, { PureComponent } from 'react'
import './app.component.sass'

export class App extends PureComponent {
  render() {
    return <h1 className="app">Hello world this is practicing</h1>
  }
}
