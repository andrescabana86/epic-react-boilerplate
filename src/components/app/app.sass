@import "src/style/colors"
.app
  color: $black