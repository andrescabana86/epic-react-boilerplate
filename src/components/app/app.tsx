import React from 'react'
import './app.sass'

export class App extends React.PureComponent {
  render() {
    return (
      <section className="app">
        <header className="app__header">
          <h1 data-testid="title" className="app__header-title">
            Welcome to my boilerplate
          </h1>
        </header>
      </section>
    )
  }
}
