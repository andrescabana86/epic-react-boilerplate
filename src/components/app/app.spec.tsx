import React from 'react'
import { render, cleanup } from '@testing-library/react'
import { App } from '@components/app/app'

describe('<App />', () => {
  afterEach(cleanup)

  it('should render app component', () => {
    const { asFragment } = render(<App />)
    expect(asFragment()).toMatchSnapshot()
  })
})
