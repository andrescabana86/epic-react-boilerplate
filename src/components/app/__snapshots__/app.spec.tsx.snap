// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render 1`] = `
<DocumentFragment>
  <section
    class="app"
  >
    <header
      class="app__header"
    >
      <h1
        class="app__header-title"
        data-testid="title"
      >
        Welcome to my boilerplate
      </h1>
    </header>
  </section>
</DocumentFragment>
`;
